openstack/compute/v2/extensions/services/requests.go

```diff
@@ -80,3 +80,12 @@ func Update(client *gophercloud.ServiceClient, id string, opts UpdateOpts) (r Up
 	_, r.Header, r.Err = gophercloud.ParseResponse(resp, err)
 	return
 }
+
+// Delete will delete the existing service with the provided ID.
+func Delete(client *gophercloud.ServiceClient, id string) (r DeleteResult) {
+	resp, err := client.Delete(updateURL(client, id), &gophercloud.RequestOpts{
+		OkCodes: []int{204},
```

Comment:
https://docs.openstack.org/api-ref/compute/?expanded=delete-compute-service-detail#delete-compute-service

I confirm this is 204.